apps/backend/src/mutations/FapMutations.ts

```diff
@@ -379,6 +380,128 @@ export default class FapMutations {
       });
   }
 
+  /**
+   * Assigns all remaining reviews needed for a FAP among its members. Attempts to distribute reviews as evenly as possible.
+   * Assumes there is ony one call in the review phase for a FAP at a time.
+   * @param agent User
+   * @param args ID of the FAP to mass assign reviews for
+   * @returns Updated FAP
+   */
+  @Authorized([Roles.USER_OFFICER, Roles.FAP_SECRETARY, Roles.FAP_CHAIR])
+  @EventBus(Event.FAP_MEMBER_ASSIGNED_TO_PROPOSAL)
+  async massAssignReviews(
```

Comment:
While this method looks like it'll work, I needed to write out a summary to figure out what it's doing, and I feel like it might be longer and more complex than it needs to be. 

At present I think it's something like this:

- Get how many proposals each reviewer is assigned to in the current round
- Get the highest number of reviews a reviewer has
- Count the total number of reviews needed for each reviewer to have that maximum
- Count the total number of reviews needed by the proposals
- Calculate how many reviews to assign to each proposer
- Assign reviews by looping over reviewers:
  - lookup from earlier how many reviews they need to give
  - looping over proposals until you've found that many that they're not already on
  - assign the reviewer to those proposals


I think a simpler process would be:
- Loop over each proposal
  - while it has fewer reviewers than needed
    - find the reviewer with the lowest number of reviews who isn't assigned to this proposal
    - assign them